test/conformance/README.md

```diff
@@ -4,6 +4,9 @@ At this point, conformance tests include matches for individual adapters
 that allow you to ignore errors from the listed tests.
 This solution allows regular execution of cts tests on GHA
 and prevents further errors.
+There is an environment variable NUMBER_OF_DEVICES that allow you to choose
```

Comment:
heh, this new sentence is put in a middle of something totally different 😉  please move it to the bottom

Reply:
I've changed it.